projects/angular/src/progress/progress-bars/_progress-bars.clarity.scss

```diff
@@ -753,28 +755,28 @@
         margin-left: 0;
         margin-right: 0;
 
-        @include utils-mixins.css-var(color, clr-progress-clr-row-color, inherit, variables.$clr-use-custom-properties);
+        @include utils-mixins.css-var(color, clr-progress-label-color, inherit, variables.$clr-use-custom-properties);
         @include utils-mixins.css-var(
           font-weight,
-          clr-progress-clr-row-font-weight,
+          clr-progress-standart-font-weight,
```

Comment:
`standard`

Reply:
Silly mistake. corrected in commit b43008a